src/lavatory/commands/purge.py

```diff
@@ -23,11 +23,13 @@
     multiple=True,
     required=False,
     help='Name of specific repository to run against. Can use --repo multiple times. If not provided, uses all repos.')
-def purge(ctx, dryrun, policies_path, default, repo):
+@click.option('--repo-type', default='local', required=False, type=click.Choice(['local', 'virtual', 'cache', 'any']),
```

Comment:
Also, since this list is mentioned twice, `['local', 'virtual', 'cache', 'any']`, maybe it should be a variable?

Reply:
added this too, started a consts.py to hold constants like that. I'm sure more uses will come up